TOC.md

```diff
@@ -381,6 +381,8 @@
       - [Data Check for Tables with Different Schema or Table Names](/reference/tools/sync-diff-inspector/route-diff.md)
       - [Data Check in the Sharding Scenario](/reference/tools/sync-diff-inspector/shard-diff.md)
       - [Data Check for TiDB Upstream and Downstream Clusters](/reference/tools/sync-diff-inspector/tidb-diff.md)
+    + TiUP
+      - [Manage Components](/reference/tools/tiup/manage-component.md)
```

Comment:
```suggestion
      - [Manage Components with Commands](/reference/tools/tiup/manage-component.md)
```
According to the last paragraph in https://github.com/pingcap/docs/pull/2482/files, this document is aimed to introduce all TiUP commands that are used to manage components. So we'd better mention the commands in the title.